test/jest-runner.js

```diff
@@ -0,0 +1,210 @@
+const path = require('path');
+const wrapper = require('@playwright/jest-wrapper');
+const DefaultTestRunner = require('../utils/testrunner');
+const {TestRunner, TestRun} = require('../utils/testrunner/TestRunner');
+const {Environment} = require('../utils/testrunner/Test');
+const utils = require('./utils');
+const fs = require('fs');
+const pirates = require('pirates');
+const babel =require('@babel/core');
+const testRunnerInfo = makeTestRunnerInfo();
+
+
+module.exports = wrapper.createJestRunner(async ({path: filePath}) => {
+  const config = require('./test.config');
```

Comment:
If we go this way, can we instead refactor existing `test.js` to expose makeTestRunnerInfo and collect, or whatever we need?

Reply:
A bunch of the code went away when environments got refactored into a new file. I can refactor some more, but what's left is really all hacks around the puppeteer test runner. I think leaving `jest-runner` ugly is more honest about the ugly things it does, like initialize a TestRunner with fake config just to access the test collector.

If we really like jest, we can put it on CI and remove the other test runner. If we don't like jest, I'm not really concerned about the quality of the code surrounding it, as its just a toy. It doesn't seem worth it to do major refactoring to the puppeteer test runner to support this.